app/Eloquent/Model.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Brewmap\Eloquent;
+
+use Brewmap\Eloquent\Traits\CamelcasedAttributes;
+use Illuminate\Database\Eloquent\Model as EloquentModel;
+
+abstract class Model extends EloquentModel
+{
+    use CamelcasedAttributes;
```

Comment:
If we agree on 'camel case' written separately, then the name of this trait should be refactored.

Reply:
Changed, good point.